scripts/post_leap.sh

```diff
@@ -35,4 +35,11 @@ if [[ ! -f "${UPGRADE_LEAP_MARKER_FOLDER}/deploy-rpc.complete" ]]; then
 else
   log "deploy-rpc" "skipped"
 fi
+
+if [ "$UPGRADE_ELASTICSEARCH"=="yes" ]; then
```

Comment:
Add a space to both sides of the ==.  Other than that LGTM.

Reply:
You're right, spaces are better.